packages/mdc-typography/_mixins.scss

```diff
@@ -35,6 +35,10 @@
   font-weight: #{map-get($style-props, font-weight)};
   letter-spacing: map-get($style-props, letter-spacing);
   line-height: map-get($style-props, line-height);
+
+  @if $style == "button" {
```

Comment:
I wouldn't *force* all our users to include this mixin all the time. I would leave *recommendation* that they include both mixins. Its a little more flexible. 

Theres nothing on the [typography spec](https://material.io/guidelines/style/typography.html#) about buttons text being centered, though there is some information about it being "all caps". But there is ALSO information about *not* using caps for certain languages. So again, not forcing the user to include the other mixin gives us flexibility to handle these situations.